DOCS-12033-adding steps to add TLS JVM args (#476)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/deploy-to-runtime-fabric.adoc

+
If needed, review xref:deploy-resource-allocation.adoc[determine resource allocation] for applications deployed to Runtime Fabric.

> === (Optional) Enable TLSv1 and TLSv1.1

Author: ```suggestion
=== (Optional) Enable TLS v1 and TLS v1.1
```

> The Mule runtime disables TLSv1 and TLSv1.1 by default. When you deploy an application to Runtime Fabric, you can enable TLSv1 or TLSv1.1 using JVM args.

Author: ```suggestion
The Mule runtime disables TLS v1 and TLS v1.1 by default. When you deploy an application to Runtime Fabric, you can enable TLS v1 or TLS v1.1 using JVM args.
```

> The Mule runtime disables TLS v1 and TLS v1.1 by default. When you deploy an application to Runtime Fabric, you can enable TLS v1 or TLS v1.1 using JVM args.

Reviewer: ```suggestion
Mule runtime disables TLS v1 and TLS v1.1 by default. When you deploy an application to Runtime Fabric, you can enable TLS v1 or TLS v1.1 using JVM args.
```

> The Mule runtime disables TLS v1 and TLS v1.1 by default. When you deploy an application to Runtime Fabric, you can enable TLS v1 or TLS v1.1 using JVM args.

Reviewer 2: ```suggestion
Starting with the August 3, 2021 patch releases, Mule runtime disables TLS v1 and TLS v1.1 by default  for all runtime versions. When you deploy an application to Runtime Fabric, you can enable TLS v1 or TLS v1.1 using JVM args.
```

> Starting with the August 3, 2021 patch releases, Mule runtime disables TLS v1 and TLS v1.1 by default for all runtime versions. When you deploy an application to Runtime Fabric, you can enable TLS v1 or TLS v1.1 using JVM args.

Author: ```suggestion
Starting with the xref:release-notes::runtime-fabric/runtime-fabric-runtimes-release-notes.adoc#august-3-2021[August 3, 2021 patch releases], Mule runtime disables TLS v1 and TLS v1.1 by default for all runtime versions. When you deploy an application to Runtime Fabric, you can enable TLS v1 or TLS v1.1 using JVM args.
```

> Starting with the August 3, 2021 patch releases, Mule runtime disables TLS v1 and TLS v1.1 by default  for all runtime versions. When you deploy an application to Runtime Fabric, you can enable TLS v1 or TLS v1.1 using JVM args.

Author: ```suggestion
Starting with the August 3, 2021 patch releases, Mule runtime disables TLS v1 and TLS v1.1 by default for all runtime versions. When you deploy an application to Runtime Fabric, you can enable TLS v1 or TLS v1.1 using JVM args.
```

> Starting with the August 3, 2021 patch releases, Mule runtime disables TLS v1 and TLS v1.1 by default  for all runtime versions. When you deploy an application to Runtime Fabric, you can enable TLS v1 or TLS v1.1 using JVM args.

Author: ```suggestion
Starting with the August 3, 2021 patch releases, Mule runtime disables TLS v1 and TLS v1.1 by default  for all runtime versions. When you deploy an application to Runtime Fabric, you can enable TLS v1 or TLS v1.1 using JVM args.
```

. Click the *JVM* tab. 

> . In the *Java Virtual Machine arguments*, enter one of the following arguments:

Author: ```suggestion
. In the *Java Virtual Machine arguments* field, add one of the following arguments:
```

> . In the *Java Virtual Machine arguments* field, add one of the following arguments:

Reviewer 2: Do we need to separately mention that these properties can also be passed using other supported deployment methods like API & Mule maven plugin?

Author: not here. I fill figure out where to add them in the other places--perhaps in the release notes
